content/news/playtest-20200323.md

```diff
@@ -0,0 +1,67 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Today's playtest constitutes our last release candidate for this cycle and adresses the issues discovered in the previous playtest.
+
+The [changes in playtest-20200323](TODO) include:
```

Comment:
We could mention the weather effects fix too

Reply:
I thought the weather effects PR was a regression fix?

Reply:
yeah, i meant to include a mention for regression fixes like this.